review/passenger-service/src/test/java/cz/muni/fi/airportmanager/passengerservice/grpc/FlightCancellationServiceTest.java

```diff
@@ -7,6 +7,7 @@
 import io.quarkus.test.junit.QuarkusTest;
 import io.quarkus.test.vertx.RunOnVertxContext;
 import io.quarkus.test.vertx.UniAsserter;
+import io.smallrye.mutiny.Uni;
```

Comment:
if it's not used, don't add it. This will be highlighted in IDEs and students will be asking.